src/api/annotate/InsertAnnotation.ts

```diff
@@ -0,0 +1,33 @@
+/* Import Dependencies */
+import axios from 'axios';
+
+/* Import Types */
+import { Annotation, AnnotationTemplate } from 'global/Types';
+
+
+const InsertAnnotation = async (annotationRecord: AnnotationTemplate, token?: string) => {
+    if (annotationRecord && token) {
+        let annotation = <Annotation>{};
+
+        const endPoint = '/annotations';
+
+        await axios({
```

Comment:
I think that the .then and the await do a similar thing. Are you sure this is correct. If I look at: https://rapidapi.com/guides/axios-async-await It is either the .then or the await

Reply:
It is not quite the same because await really waits for the call to complete before anything in the function can be executed, .then does execute when the result is reeled in, but the other stuff in the function will continue. Chose await over .then for the API calls because of the waiting, think this is important. Removed .then for all api calls and introduced try/catch for checking if the fetch was successful or not.